src/sqlfluff/dialects/dialect_exasol_fs.py

```diff
@@ -122,6 +122,11 @@ class FileSegment(BaseSegment):
         allow_trailing=True,
     )
 
+    @property
+    def file_path(self):
+        """File path of a parsed SQL file."""
+        return self._file_path
+
```

Comment:
Won't this automatically be inherited from `ansi`?

```suggestion
```

Reply:
Ah no it won't because you explicitly overwrite it. Can we set it to copy from ANSI version so not repeating the code?

Reply:
I think it needs to be overwritten because scripts not delimited by `;` Instead they are delimited by `/`. But maybe there is another way?

Currently I added a new class `FileSegment` as you suggested.